ls1intum/Artemis
# Redirect specific exercises routes to the shared overview

### Checklist

### Motivation and Context

Having views only for specific exercise types doesn't make much sense at the moment (no extra information) and makes navigation harder since you can't go to all exercises from a single exercise using the breadcrumbs.

### Description

Redirects the specific routes to the general route.

### Steps for Testing

1. Open any site belonging to a specific exercise type, e.g. "Create a new Text Exercise" (can also be edit, scores, ...).
2. Click on `text-exercises` (or `modeling-exercises` etc. depending on which type you chose).
3. Check that you were redirected to the overview over all exercise types

Repeat for each of the 5 types.

Example:
Clicking on `text-exercises` here:
![grafik](https://user-images.githubusercontent.com/72132281/104251003-1546da00-546f-11eb-8b47-4d4e3c62da68.png)
Goes to the `/course-management/209/exercises` URL.
